### DISCUSSION TITLE
What is this binary called userfeedbackconsole?

### DISCUSSION THREAD

**Discussion by way2me** [prasanthrangan/hyprdots#1883] [General]
I have a general question for which I don't have an answer right now

There is an app in the application menu called UserFeedbackConsole. Where is it hosted on github? Who is maintaining it? Is it related to this repository or about any other app/service?

**Comment by kRHYME7** [comment]
dolphin 

![image](https://github.com/user-attachments/assets/0b1d213a-2bbb-410d-9660-e3cec92a6fc2)

**Comment by way2me** [comment]
Thanks buddy
